packages/inputs-minimal/src/textfields_meta/index.jsx

```diff
@@ -6,19 +6,22 @@ const Meta = ({
   error,
   hint,
   className,
-}) => (
-  <div className={clsx(styles.meta, className)}>
-    {error && (
+}) => {
+  const hasHintOrError = Boolean(hint || error);
+  const useInputGroup = hasHintOrError && typeof error !== 'string';
```

Comment:
`input-minmal` has no idea what an 'input group' is. It doesn't exist in the package. Maybe just go with ...

```js
const hasError = typeof error === 'string';
const hasHint = Boolean(hint);

if (!hasError && !hasHint) return null;
```

Reply:
Yeah, that makes sense.